baselines/dom.generated.d.ts

```diff
@@ -3465,6 +3466,9 @@ interface Element extends Node, GlobalEventHandlers, ElementTraversal, NodeSelec
     readonly scrollWidth: number;
     readonly tagName: string;
     innerHTML: string;
+    readonly assignedSlot: HTMLSlotElement | null;
```

Comment:
As I read the spec, in a browser that implements this API, `assignedSlot` will never be `undefined`.

It's true that in an environment without the API it could be `undefined`, but this is true of most browser APIs.